### PR TITLE
Update contributing.md

### PR DESCRIPTION
**Pull request by MichaelABroadcom**
Signed-off-by: Michael Aimino <michael.aimino@broadcom.com>

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Made an update to contributing.md. The first PR is created is mangled, so I deleted it and created this PR.

:heart:Thank you!

### CODE REVIEW

File: docs/contribute/contributing.md [hunk 1]
```diff
@@ -116,14 +116,13 @@ Style:
 
     Example: _myHost_
 
-- If wrap using angle brackets `<>` within code examples, italic font is not supported.
+- If wrap using angle brackets <> within code examples, italic font is not supported.
```
Comment (covalentbond) [inline docs/contribute/contributing.md:119]: Hey, can you revert this change?

If we will are writing plain `<>`. It will most probably throw a Compilation Error saying "Expected corresponding JSX closing tag for <>". 

It tries to render `<>` as a component, instead of a text & any component without closing bracket causes an Error.

File: docs/contribute/contributing.md [hunk 2]
```diff
@@ -343,7 +342,7 @@ Use their English equivalents instead. Latin abbreviations are sometimes misunde
 Latin  | English equivalent
 ---|---
 e.g.  | for example
-  etc. | and so on. <br/> When you list a clear sequence of elements such as "1, 2, 3, and so on" and "Monday, Tuesday, Wednesday, and so on." Otherwise, rewrite the sentence to replace "etc." with something more descriptive such as "and other output."
+  etc. | and so on. <br> When you list a clear sequence of elements such as "1, 2, 3, and so on" and "Monday, Tuesday, Wednesday, and so on." Otherwise, rewrite the sentence to replace "etc." with something more descriptive such as "and other output."
```
Comment (covalentbond) [inline docs/contribute/contributing.md:345]: This one also needs to be reverted.

All the HTML elements must have a closing tag like `<br/>`, ``, unless they would throw Compilation Error.

File: docs/contribute/contributing.md [hunk 3]
```diff
@@ -364,15 +363,15 @@ application  | app
...
+later  | higher <br> Do not use to describe versions of software or fix packs.
 macOS| MacOS
-Node.js  | node.js <br/> Nodejs
+Node.js  | node.js <br> Nodejs
 plug-in| plugin
 REXX  | Rexx
-UNIX System Services <br/> z/OS UNIX System Services  | USS
-zLUX  | ZLUX <br/> zLux
+UNIX System Services <br> z/OS UNIX System Services  | USS
+zLUX  | ZLUX <br> zLux
```
Comment (covalentbond) [inline docs/contribute/contributing.md:376]: Similarly in these!

**Comment by covalentbond** [comment]
I'm actually planning to setup a Netlify preview link against the `docs-staging` branch.  So that with every PR - a Netlify link will be generated similar to what's generated against the `master` branch.

If the build fails. It will probably throw an error & will help to debug fast. :)

**Comment by nannanli** [comment]
@covalentbond Thank you for reviewing and addressing the issues! For Netlify, I'll work with John to set it up for the `docs-staging` branch using the OMP Netlify account. : )
